features/admin/product/managing_product_variants/sorting_product_variants_within_product_by_position.feature

```diff
@@ -12,43 +12,43 @@ Feature: Sorting listed product variants from a product by position
         And this product has also an "Opel Insignia Sedan" variant at position 1
         And I am logged in as an administrator
 
-    @todo @ui @api
+    @ui @api
     Scenario: Product variants are sorted by position in ascending order by default
         When I view all variants of the product "Opel Insignia"
         Then I should see 3 variants in the list
         And the first variant in the list should have name "Opel Insignia Hatchback"
         And the last variant in the list should have name "Opel Insignia Sports Tourer"
 
-    @todo @ui @api
+    @ui @api
     Scenario: Sorting product variants in descending order
         When I view all variants of the product "Opel Insignia"
         And I start sorting variants by position
         Then the first variant in the list should have name "Opel Insignia Sports Tourer"
         And the last variant in the list should have name "Opel Insignia Hatchback"
 
-    @todo @ui @api
+    @ui @api
     Scenario: New product variant with no position is added as the last one
         Given the product "Opel Insignia" has also an "Opel Insignia Country Tourer" variant
         When I view all variants of the product "Opel Insignia"
         Then I should see 4 variants in the list
         And the last variant in the list should have name "Opel Insignia Country Tourer"
 
-    @todo @ui @api
+    @ui @api
     Scenario: New product variant with position 0 is added as the first one
         Given the product "Opel Insignia" has also an "Opel Insignia Country Tourer" variant at position 0
         When I view all variants of the product "Opel Insignia"
         Then I should see 4 variants in the list
         And the first variant in the list should have name "Opel Insignia Country Tourer"
 
-    @todo @ui @mink:chromedriver @api
+    @ui @mink:chromedriver @api
     Scenario: Setting product variant as the first one in the list
         When I view all variants of the product "Opel Insignia"
         And I set the position of "Opel Insignia Sedan" to 0
         And I save my new configuration
```

Comment:
I'm a little distracted by the term "configuration" because it means more than just an order, when really we're just submitting a new order.
```suggestion
        And I save my new order
```
or
```suggestion
        And I save my new elements order
```
And such a change could be reflected in Behat function names, selectors, elements, etc.
Wdyt?